api/bin/www

```diff
@@ -4,7 +4,7 @@ const app = require('../src/app');
 const debug = require('debug')('api:server');
 const http = require('http');
 
-const port = normalizePort(process.env.PORT || '9000');
+const port = normalizePort(process.env.PORT || '4000');
```

Comment:
Remove this change, let's keep it 9000 - usually backend has a higher port number than the frontend server